kv/txn_correctness_test.go

```diff
@@ -178,28 +183,34 @@ func scanCmd(c *cmd, txn *client.Txn, t *testing.T) *roachpb.Error {
 	return nil
 }
 
-// incCmd adds one to the value of c.key in the env and writes
-// it to the db. If c.key isn't in the db, writes 1.
+// incCmd adds one to the value of c.key in the env (as determined by
+// a previous read, or else assumed to be zero) and writes it to the
+// db. If c.key isn't in the db, writes 1.
 func incCmd(c *cmd, txn *client.Txn, t *testing.T) *roachpb.Error {
-	r, pErr := txn.Inc(c.getKey(), 1)
-	if pErr != nil {
+	r := c.env[c.key] + 1
+	if pErr := txn.Put(c.getKey(), r); pErr != nil {
 		return pErr
 	}
-	c.env[c.key] = r.ValueInt()
-	c.debug = fmt.Sprintf("[%d]", r.ValueInt())
+	c.env[c.key] = r
+	c.debug = fmt.Sprintf("[%d]", r)
 	return nil
 }
 
-// sumCmd sums the values of all keys != c.key read during the transaction and
-// writes the result to the db.
-func sumCmd(c *cmd, txn *client.Txn, t *testing.T) *roachpb.Error {
+// writeCmd sums values from the env (and possibly numeric constants)
+// and writes the value to the db. "c.key" here needs to be parsed in
+// the context of this command, which is the destination key followed
+// by a command and then a "+"-separated list of keys from the env or
```

Comment:
s/nd// (or actually, it looks like this comment has rotted and should describe the use of key and endKey)